plugins/item_tasks/web_client/views/SelectTaskView.js

```diff
@@ -0,0 +1,66 @@
+import SearchFieldWidget from 'girder/views/widgets/SearchFieldWidget';
+import View from 'girder/views/View';
+
+import 'girder/utilities/jquery/girderEnable';
+import 'girder/utilities/jquery/girderModal';
+
+import SelectTaskViewDialogTemplate from '../templates/selectTaskViewDialog.pug';
+import selectTaskViewDescriptionTemplate from '../templates/selectTaskViewTargetDescription.pug';
+
+import '../stylesheets/selectTaskView.styl';
+
+/**
+ * A dialog for creating tasks from a specific item.
+ */
+var SelectTaskView = View.extend({
+    events: {
+        'submit #g-select-task-form': function (e) {
+            e.preventDefault();
+            this.$('.g-submit-select-task').girderEnable(false);
+
+            this.trigger('g:selected', {
+                itemId: this.item.id,
+                taskId: this.taskId
+            });
+        }
```

Comment:
This modal should close itself here. We shouldn't necessarily assume that the user of this view will perform a redirection after a selection occurs.